### ISSUE TITLE
[BUG] "Could not resolve reference to DataModelField" when using abstract field in future()

### ISSUE THREAD

**Issue by onimitch** [zenstackhq/zenstack#1695]
**Description and expected behavior**
```
abstract model SoftDelete {
    deleted Int @default(0) @omit
}

model MyModel extends SoftDelete {
    id      String @id @default(cuid())
    name    String

    @@deny('update', deleted != 0 && future().deleted != 0)
}
```

The above @deny rule errors: `Could not resolve reference to DataModelField named 'deleted'.`

Workaround for now: Place the deny rule in the abstract model.

**Environment (please complete the following information)**

-   ZenStack version: 2.5.1
-   Prisma version: 5.18.0
-   Database type: Postgresql

**Comment by ymc9** [comment]
Fixed in 2.6.0

**Comment by onimitch** [comment]
@ymc9 thanks!
